### DISCUSSION TITLE
Definitions are not available in catalogLibrary.

### DISCUSSION THREAD

**Discussion by indubabu** [KaotoIO/kaoto#1213] [Development]
I have taken update from recent changes and  run the  app  with "yarn workspace @kaoto/kaoto run start".  while fetching camel-catalog in runtimeProvider.tsx file - catalogLibrary.definitions  is throwing undefined error. catalogLibrary has only catalogs and schemas and missing the definitions. Kindly guide me, if I have missed any step.

**Comment by lhein** [comment]
try in this order:
1. yarn install
2. yarn workspace @kaoto/camel-catalog run build
3. yarn workspace @kaoto/kaoto run start        

That works fine for me.
